### PR TITLE
Added story to show well in intersection view

### PR DESCRIPTION
**Pull request by shadab-skhan**

### CODE REVIEW

File: react/src/lib/components/DeckGLMap/layers/intersection/unfoldedGeoJsonLayer.ts [hunk 1]
```diff
@@ -15,11 +17,10 @@ function getUnfoldedPath(object: Feature) {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const a: any[] = [];
     delta.forEach((d) => {
-        const prev = a.at(-1) || 0;
+        const prev = a.at(-1) || first_x;
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/intersection/unfoldedGeoJsonLayer.ts:20]: World coordinates don't make sense in this projection. Instead X should mean distance from the beginning of the fence. We should change the axis origin in this viewport 0.

File: react/src/lib/components/DeckGLMap/layers/intersection/unfoldedGeoJsonLayer.ts [hunk 2]
```diff
@@ -15,11 +17,10 @@ function getUnfoldedPath(object: Feature) {
...
         a.push(d + prev);
     });
-    const planeY = 2000;
-    const vAbscissa = zip(a, [...a].fill(planeY), z);
+    const vAbscissa = zip(a, y, z);
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/intersection/unfoldedGeoJsonLayer.ts:23]: What is the purpose changing the y here?

**Review by shruthirai — APPROVED** [review]

File: react/src/lib/components/DeckGLMap/layers/intersection/unfoldedGeoJsonLayer.ts (hunk 1 above)
Comment (shadab-skhan) [reply react/src/lib/components/DeckGLMap/layers/intersection/unfoldedGeoJsonLayer.ts]: Changed back to 0

**Comment by shadab-skhan** [comment]
fix #1068

**Pull request merged**